skin/danoni_skin_default.css

```
@@ -70,6 +70,10 @@
 	--settings-slice-gaugeTable-x: 1;
 	--settings-border-gaugeTable-x: 1px;
 
+	/* 譜面明細子画面: ハイスコア表示 */
+	/*--settings-assist-x: var(--common-kita); */
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)